### PR TITLE
[JIT] Add support for list()

### PR DESCRIPTION
**Pull request by eellison**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#33818 [JIT] Add support for list()**
* #33783 [JIT] add support for torch.norm

Differential Revision: [D20121915](https://our.internmc.facebook.com/intern/diff/D20121915)

### CODE REVIEW

**Comment by eellison** [comment]
There are a few bound `aten::list` ops. If we had a backwards compat ast-ast writer we could rewrite `torch.list` -> `list` and remove them.

File: test/test_jit.py [hunk 1]
```diff
@@ -10032,6 +10032,12 @@ def forward(self, x, skip_name):
             self.checkModule(M(), (inp, name))
             self.checkModule(M2(), (inp, name))
 
+    def test_list_keyword(self):
+        def foo():
```
Comment (driazati) [inline test/test_jit.py]: Can you add a check for a bad input as well? e.g. that `list(1)` says something like Python's `TypeError: 'int' object is not iterable`

**Review by driazati — APPROVED** [review]

File: test/test_jit.py (hunk 1 above)
Comment (eellison) [reply test/test_jit.py]: there are tests that already do that like `test_sum_list_wrong_type`

**Comment by mrshenli** [comment]
`pytorch_linux_xenial_cuda10_1_cudnn7_py3_slow_test` on master starts to time out on this PR. It actually also fails on this PR but passed on the one below it (#33783). Reverting. 

```
Mar 05 07:04:32 Generating XML reports...
Mar 05 07:04:33 Running test_jit ... [2020-03-05 07:04:33.071318]
Mar 05 07:04:36 
Mar 05 07:04:36 Running tests...
Mar 05 07:04:36 ----------------------------------------------------------------------

Too long with no output (exceeded 1h30m0s): context deadline exceeded
```

**Comment by eellison** [comment]
Are you sure that’s related ?

**Pull request closed without merging**